witness/src/Dscp/DB/Rocks/Real/Functions.hs

```diff
@@ -1,27 +1,56 @@
 {-# LANGUAGE TypeApplications #-}
 
 module Dscp.DB.Rocks.Real.Functions
-       ( -- * Closing/opening
+       (
+         -- * Closing/opening
          openRocksDB
        , closeRocksDB
        , openNodeDB
        , closeNodeDB
+
          -- * Reading/writing
        , rocksGetBytes
        , rocksPutBytes
        , rocksDelete
+       , rocksWriteBatch
+
+       , iterFoldl
+
+       , getDB
+
+         -- * More high-level interface
+       , DeserialisationError
+       , get
+       , put
+       , delete
        ) where
 
+import Prelude hiding (get, put)
```

Comment:
What is this thing for?

Reply:
We're exporting `put` and `get` from that module.

Reply:
Yes, but why would you name a function `put` or `get` given that you know it would clash with the common MTL-related terminology? :thinking:

Reply:
Because we `get` from RocksDB and `put` things into it and the module is used qualified.
We, like have proper modules in the language.